Testing/functional/tests/cypress/integration/ui/pages/LandingPage.js

```diff
@@ -118,6 +118,12 @@ describe("Landing Page", () => {
     });
 
     it("Validate active tiles", () => {
+        cy.enableModules("");
```

Comment:
This enabling and disabling of the module within a single test works without logging out and logging back in?

Reply:
`enableModules` sets up an intercept on the call to retrieve the config, so a refresh would be required in between uses.

Nit: I think `cy.enableModules([])` better conveys the intent of enabling no modules over `cy.enableModules("")`.

At some point I think we should refactor the `enableModules` command to make it so the parameter has to be an array rather than an array or a string.